src/bin/bors.rs

```diff
@@ -109,7 +109,8 @@ fn try_main(opts: Opts) -> anyhow::Result<()> {
     }
 
     let ctx = BorsContext::new(CommandParser::new(opts.cmd_prefix), Arc::new(db), repos);
-    let (repository_tx, global_tx, bors_process) = create_bors_process(ctx, client, team_api);
+    let (repository_tx, global_tx, _mergeable_queue_tx, bors_process) =
```

Comment:
I think it's high time to return a struct with named fields here, a four tuple is becoming too messy, especially since some of these variables have the same type and could be swapped inadvertedly.